### PR TITLE
Fix fps output

### PR DESCRIPTION
**Pull request by egorovdanil**

### CODE REVIEW

File: samples/sample_encode/include/pipeline_encode.h [hunk 1]
```diff
@@ -29,6 +29,8 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 
 #include "sample_utils.h"
 #include "base_allocator.h"
+
+#define TIME_STATS 1
```
Comment (dmitryermilov) [inline samples/sample_encode/include/pipeline_encode.h]: @egorovdanil , I don't understand what change. Can you please explain how this define helps?

Comment (egorovdanil) [reply samples/sample_encode/include/pipeline_encode.h]: To output FPS, in the `pipeline_encode.cpp` it checks if a variable is defined:
`#ifdef TIME_STATS
        mfxF64 ProcDeltaTime = m_statOverall.GetDeltaTime() - m_statFile.GetDeltaTime() - m_TaskPool.GetFileStatistics().GetDeltaTime();
        msdk_printf(MSDK_STRING("Encoding fps: %.0f\n"), m_FileWriters.first->m_nProcessedFramesNum / ProcDeltaTime);
#endif`

Comment (dmitryermilov) [reply samples/sample_encode/include/pipeline_encode.h]: @AntonGrishin , I know you've recently fixed some issue with FPS reporting in samples. Was it a fix for sample_encode?

Comment (AntonGrishin) [reply samples/sample_encode/include/pipeline_encode.h]: @dmitryermilov I only enabled fps report in smt, but I didn`t touch sample encode.

Comment (dmitryermilov) [reply samples/sample_encode/include/pipeline_encode.h]: Okay. For it such conditional compilation doesn't makes  sense.  @egorovdanil , please move this fps reporting outside of #ifdef

Comment (egorovdanil) [reply samples/sample_encode/include/pipeline_encode.h]: done

**Comment by dmitryermilov** [comment]
@egorovdanil , please rename commit. Now the name doesn't match code changes.

**Pull request merged**